.github/workflows/webviz-config.yml

```diff
@@ -44,6 +44,7 @@ jobs:
 
       - name: 📦 Install webviz-config with dependencies
         run: |
+          pip install urllib3==1.24.3
```

Comment:
Is pinning urllib3==1.24.3 necessary?

Reply:
Yes, otherwise the build will break. It think this was suggested by @anders-kiaer .